crates/bevy_render/src/view/window.rs

```diff
@@ -167,9 +167,14 @@ pub fn prepare_windows(
             height: window.physical_height,
             usage: wgpu::TextureUsages::RENDER_ATTACHMENT,
             present_mode: match window.present_mode {
+                #[allow(deprecated)]
```

Comment:
Fifo is the only guaranteed-to-work fundamental present mode. I don't think we should deprecate it.